test/cypress/integration/03-expenses.page.test.js

```diff
@@ -1,3 +1,155 @@
+    cy.get('input[name="items[1].description"]').type('Potatoes for the giant raclette');
+    cy.get('input[name="items[1].amount"]').type('{selectall}2.50');
+    cy.getByDataCy('expense-summary-btn').click();
+
+    // Submit!
+    cy.getByDataCy('submit-expense-btn').click();
+    cy.contains('[data-cy="temporary-notification"]', 'Expense submited!');
+    cy.contains('[data-cy="expense-page-content"]', 'Brussels January team retreat');
+    cy.getByDataCy('dismiss-temporary-notification-btn').click();
+    cy.getByDataCy('temporary-notification').should('not.exist');
+
+    // Do this again for second expense
+    cy.wait(300);
+    cy.visit(`/new-collective${collectiveSlug}/expenses/new/v2`);
+    cy.wait(200);
+    cy.getByDataCy('radio-expense-type-RECEIPT').click();
+    cy.get('input[name="description"]').clear().type('May Invoice');
+    cy.getByDataCy('styled-input-tags-open').click();
+    cy.getByDataCy('styled-input-tags-input').type('second{enter}{enter}');
+    cy.getByDataCy('styled-input-tags-open').click();
+    cy.getByDataCy('styled-input-tags-input').type('all{enter}{enter}');
+
+    // Upload 2 files to the multi-files dropzone
+    cy.fixture('images/receipt.jpg').then(fileContent => {
+      const getFile = idx => ({ fileContent, fileName: `receipt${idx}.jpg`, mimeType: 'image/jpeg' });
+      const files = [getFile(1), getFile(2)];
+      cy.getByDataCy('expense-multi-attachments-dropzone').upload(files, { subjectType: 'drag-n-drop' });
+    });
+    cy.getByDataCy('expense-attachment-form').should('have.length', 2);
+    cy.getByDataCy('expense-summary-btn').should('be.disabled');
+
+    // Fill info for first attachment
+    cy.get('input[name="items[0].description"]').type('Fancy restaurant');
+    cy.get('input[name="items[0].amount"]').type('{selectall}123');
+    cy.getByDataCy('expense-summary-btn').should('be.disabled');
+    cy.get('input:invalid').should('have.length', 2); // Missing attachment desctiption+amount
+    cy.getByDataCy('expense-items-total-amount').should('contain', '--.--'); // amount for second item is missing
+
+    // Select Payout Method
+    cy.getByDataCy('payout-method-select').click();
+    cy.contains('[data-cy="select-option"]', 'New custom payout method').click();
+    cy.get('[data-cy="payout-other-info"]').type('A simple thanks would work');
+    // cy.get('input[name="privateMessage"]').type('A simple thanks would work');
+    cy.get('input[name="items[1].description"]').type('Potatoes for the giant raclette');
+    cy.get('input[name="items[1].amount"]').type('{selectall}12.50');
+    cy.getByDataCy('expense-summary-btn').click();
+
+    // Submit!
+    cy.getByDataCy('submit-expense-btn').click();
+    cy.contains('[data-cy="temporary-notification"]', 'Expense submited!');
+    cy.contains('[data-cy="expense-page-content"]', 'May Invoice');
+    cy.getByDataCy('dismiss-temporary-notification-btn').click();
+    cy.getByDataCy('temporary-notification').should('not.exist');
```

Comment:
This is a lot of code for just creating the expense. I understand that this was copy-pasted from the other test, but here we don't want to re-test the whole flow, we just want to create an expense.

Can this be simplified?

Reply:
Done